packages/server-wallet/src/protocols/actions.ts

```diff
@@ -7,9 +7,9 @@ import {Bytes32} from '../type-aliases';
 Actions that protocols can declare.
```

Comment:
In general the types in this file seem important for the wallet and not specific to "protocol actions", maybe there should be a `src/types.ts` file which includes only these types to make it easier to find things.

Reply:
Yup, definitely worth thinking about where these actions should be defined.

My reasoning is that these protocols are actually code that should be shared between the server wallet and the browser wallet. So, the actions that protocols declare should be separate from the server wallet's own types.